### ISSUE TITLE
Returned palette doesn't include source color

### ISSUE THREAD

**Issue by haxmart** [bobthered/tailwindcss-palette-generator#6]
Source color should be included somewhere within the returned palette - keen to use this within my project but can foresee user frustration of not being returned the specific value they've given (e.g. if they're working with strict brand guidelines)

**Comment by Yard-Daniel** [comment]
@maxhart7 ran into the same issue. 

Expected the source color to be set as color-500 like this https://javisperez.github.io/tailwindcolorshades/?strikemaster=8a6086

**Comment by jhgeluk** [comment]
Experiencing same issue

**Comment by bobthered** [comment]
Thanks for the feedback.  This issue is actually by design.  The goal of the palette generator is to give you a full palette to choose from, based off the hue & saturation of the supplied color.  The lightness / shades are then calculated based on predefined values ( [which also can be overwritten](https://github.com/bobthered/tailwindcss-palette-generator/tree/main#example-3) ).  If the source color was preserved, you could end up with crowding.

Let's take a look at two examples:

<ins>Light Color</ins>
`#ffffff`
If we preserved the white color for our `500` shade, then everything from shade `50 - 500` would be the same.  You can see this: https://javisperez.github.io/tailwindcolorshades/?white=FFFFFF

<ins>Dark Color</ins>
`#000000`
If we preserved the black color four our `500` shade, then everything from shade `500 - 950` would be the same.  You can see this:
https://javisperez.github.io/tailwindcolorshades/?black=000000

One option I did explore was finding the shade with the closest lightness value, and inserting it into the palette.  The two examples would be something like:
```
// #ffffff
{
    '50': '#ffffff',
    '100': '#f2f2f2',
    '200': '#e6e6e6',
    '300': '#d1d1d1',
    '400': '#a3a3a3',
    '500': '#757575',
    '600': '#545454',
    '700': '#3d3d3d',
    '800': '#242424',
    '900': '#121212',
    '950': '#0a0a0a'
  }

// #000000
{
    '50': '#fafafa',
    '100': '#f2f2f2',
    '200': '#e6e6e6',
    '300': '#d1d1d1',
    '400': '#a3a3a3',
    '500': '#757575',
    '600': '#545454',
    '700': '#3d3d3d',
    '800': '#242424',
    '900': '#121212',
    '950': '#000000'
  }
```

**Comment by bobthered** [comment]
I added another option `preserve` that will find the closest shade and insert the originally supplied hex in at that place.  This is set to `true` by default.  This is available in [3.2.0](https://github.com/bobthered/tailwindcss-palette-generator/tree/v3.2.0)
